ci/run-docker.sh

```diff
@@ -30,6 +30,7 @@ run() {
       --env NORUN \
       --env RUSTFLAGS \
       --env STDARCH_TEST_NORUN \
+      --env PROFILE \
```

Comment:
Without this line the debug CI wasn't actually doing anything different to the release CI and it turns out quite a few platforms are broken in debug mode. Will keep digging.

Reply:
There's some handwritten arm intrinsics that have not been fixed by https://github.com/rust-lang/stdarch/pull/1503, e.g. https://github.com/rust-lang/stdarch/blob/90914540cb3d5f48abecd1619dfc5425b76f1c8d/crates/core_arch/src/arm/neon.rs#L29

The wasm failure is due a hardcoded path to `target/release`. Not sure about freebsd, haven't looked into it yet.